tools/webdriver/webdriver/bidi/modules/network.py

```diff
@@ -41,7 +41,9 @@ def __init__(self, pattern: str):
 class Network(BidiModule):
     @command
     def add_intercept(
-        self, phases: List[str], url_patterns: Optional[List[URLPattern]] = None
+            self,
+            phases: List[str],
+            url_patterns: Optional[List[URLPattern]] = None
```

Comment:
I think we already discussed about formatters, but it would really help if we all used the same. Typically we are using black in Firefox and we will keep having formatting changes in diffs when we work on the same files (or we will stop formatting which is not great either).

Don't have a strong opinion on which formatter to use but we should settle on one.

Reply:
Indeed, it would be great to achieve consistency. We are using `yapf 0.31.0` in Chromium and in the chromium BiDi Mapper. We keep using `yapf` here for consistency and because it's easier for us as it's already installed and configured in our environments.

In principle I have no objections to switching to `black`. Is that the formatter you use in other Mozilla repositories?

This should be discussed separately though. :-) 
For now: I am only formatting the files I am touching, to try to minimize the diff noise.

Reply:
Filed https://github.com/web-platform-tests/wpt/issues/42710.

I used this comment as an example of bikeshedding in the FR, to be clear, your comment is totally reasonable, I hope it doesn't sound harsh :-)

Reply:
not at all :) And I should have made it clear that my comment was just a call for a discussion, and not something directly related to the PR